tests/rptest/services/datalake/query_engine/databricks_sql.py

```diff
@@ -59,3 +60,39 @@ def optimize_parquet_files(self, namespace: str, table: str) -> None:
             f"OPTIMIZE {self.escape_identifier(namespace)}.{self.escape_identifier(table)}"
         )
         self.logger.debug(f"OPTIMIZE result: {result}")
+        raise NotImplementedError(
+            "DatabricksSQL optimize_parquet_files is not implemented yet")
+
+    def verify_table_contents(
```

Comment:
This method does not belong to the databricks_sql service. The verifier is separate functionality which depends on a query engine/maybe databricks specific query engine.

Let's separate concerns: client/sql service vs verifier.

Reply:
You also might be interested in looking at tests/rptest/tests/datalake/datalake_verifier.py.